source/features/list-prs-for-file.tsx

```diff
@@ -106,9 +106,8 @@ const getPrsByFile = cache.function('files-with-prs', async (): Promise<Record<s
 });
 
 async function getCurrentPath(): Promise<string> {
-	// `[aria-label="Copy path"]` on blob page, `#blob-edit-path` on edit page
-	const element = await elementReady('[aria-label="Copy path"], #blob-edit-path');
-	return element!.getAttribute('value')!;
+	const viewData = JSON.parse((await elementReady('[data-target="react-app.embeddedData"]'))!.innerText.trim());
+	return viewData.payload.path;
```

Comment:
For readability I would put the JSON.parse here

```suggestion
	return JSON.parse(viewData).payload.path;
```